product_secondary_unit/models/__init__.py

```diff
@@ -1,4 +1,5 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from . import product
```

Comment:
And sort it alphabetically

```suggestion
from . import product_product
```